buzzmobile/plan/controller/src/controller.cpp

```diff
@@ -94,16 +94,19 @@ void handleDrive(const sensor_msgs::Joy::ConstPtr& joy) {
     //Motor control
     float speed = 0.0;
 
+    double maxSpeed;
+    ros::param::get("max_speed", maxSpeed); // Get max speed from constants.yaml
+    
     ROS_INFO("Last brake %d", lastBrake);
     if (!lastBrake) {
         if (joy->reverse_button) {
             float correction = -1; // Normally ranges from 1 to -1. Correct so it ranges from 0 to -2
             // Division by 2 brings to range -1 to 0. It can then be multiplied by max speed
-            speed = ((correction + joy->velocity_trigger) / 2.0) * maxFwdSpeed;
+            speed = ((correction + joy->velocity_trigger) / 2.0) * maxSpeed;
```

Comment:
Since we're not using the `maxFwdSpeed` var [1] can you delete it from this file?

[1] https://github.com/gtagency/buzzmobile/pull/119/files#diff-1cc9e5972481a641a05b751e8b9d7604R23